bitnami/redis/templates/replicas/application.yaml

```diff
@@ -503,6 +501,7 @@ spec:
           hostPath:
             path: /sys
         {{- end }}
+        {{- if and .Values.replica.containerSecurityContext.enabled .Values.replica.containerSecurityContext.readOnlyRootFilesystem }}
```

Comment:
This doesn't take into account `.Values.metrics.containerSecurityContext.enabled` and `.Values.metrics.containerSecurityContext.readOnlyRootFilesystem`